Update the description of `onabort` (#16311)
mdn/content: files/en-us/web/api/globaleventhandlers/onabort/index.md

  - Window
browser-compat: api.GlobalEventHandlers.onabort
---
{{ApiRef("HTML DOM")}}

The **`onabort`** property of the {{domxref("GlobalEventHandlers")}} mixin is the [event handler](/en-US/docs/Web/Events/Event_handlers) for processing `abort` events.

> Currently, only {{domxref("HTMLAudioElement")}} and {{domxref("HTMLVideoElement")}} interfaces (which inherit the {{domxref("HTMLMediaElement")}} interface) fire the {{domxref("HTMLMediaElement/abort_event", "abort")}} event.

Reviewer: ```suggestion
Currently, only the {{domxref("HTMLAudioElement")}} and {{domxref("HTMLVideoElement")}} interfaces (which inherit the {{domxref("HTMLMediaElement")}} interface) fire the {{domxref("HTMLMediaElement/abort_event", "abort")}} event.
```

> > **Note:** Previously the `abort` event is fired on `Window`, but it has been removed from the standard (see [HTML issue #3525](https://github.com/whatwg/html/issues/3525)).

Reviewer: ```suggestion
> **Note:** Previously the `abort` event was fired on `Window`, but has since been removed from the standard (see [HTML issue #3525](https://github.com/whatwg/html/issues/3525)).
```
